### ISSUE TITLE
Save checkpoints more often

### ISSUE THREAD

**Issue by oliverwm1** [ai2cm/ace#159]
Currently we only save the "best" checkpoint (determined by lowest val loss) and the last checkpoint. This has been fine to-date, but we are now seeing behavior where long rollout performance can get significantly worse with more training. So it would be helpful to save checkpoints every N epochs. Probably something like every 5 or 10 epochs would be good. Note the checkpoints are large (~ 2GB) for most of our models, so I don't think we should save after every epoch.
